docs/git-branching-strategies/githubflow.md

```diff
@@ -3,14 +3,14 @@ GitHub Flow is a simple and effective branching strategy which the folks at GitH
 
 GitHub Flow is in a nutshell:
 
-1) Update master to latest [upstream](../reference/git-setup.md#upstream) code
-2) Create a feature branch `git checkout -b myFeatureBranch`
-3) Do the feature/work
-4) Push feature branch to [origin](../reference/git-setup.md#origin)
-5) Create pull request from origin/<featureBranch> -> upstream/master
-6) Review, fix raised comments, merge your PR or even better, get someone else to.
+1) Update master to latest [upstream](../reference/git-setup.md#upstream) code  
```

Comment:
This should actually be `1.`

We could change to
```
1. Update master to latest [upstream](../reference/git-setup.md#upstream) code
1. Create a feature branch `git checkout -b myFeatureBranch`
```

If you use all 1's then it automatically incremented. Did you want to change this or I can merge and fix later because this is an improvement :+1: